.binder/environment.yml

```diff
@@ -1,13 +1,19 @@
 name: 67a1e320-7c47-4ea9-8df8-e868326bc90b
 channels:
   - conda-forge
-  - defaults
+  - nodefaults
 dependencies:
+  - cf-units
   - libnetcdf
+  - netcdf4=1.6.0
   - pip
   - python<3.12
   - "tensorflow<2.16=cpu*"
   - pip:
-      - icenet @ git+https://github.com/icenet-ai/icenet@3654dc4954eca6d28e16b4876bd6538abd1f0c06
       - ecmwflibs
-      - netCDF4==1.6.0
+      - icenet @ git+https://github.com/icenet-ai/icenet@32558347fa8459a4414b5f30af16b04540ba9101
+      - ipykernel
+      - jupyterlab
+      - nbdime
+      - notebook
+      - copernicusmarine
```

Comment:
These dependencies are available on conda-forge, so you can put them directly under `dependencies` instead of under `dependencies: pip:`.

Reply:
Updated as requested.